python/ml4ir/base/tests/test_monte_carlo_scorer.py

```diff
@@ -0,0 +1,97 @@
+import unittest
```

Comment:
@mohazahran sorry, let's not add more end-to-end tests, it is really slowing the build and tf.keras models are leading to memory leaks. Can you test just MC scorer by mocking everything else out using number of calls made to `call()` or `make_pred`?

Reply:
@lastmansleeping the scorer need to init everything else basically. So the only part we will be saving is creating the dataset, which we can keep it to minimum.

Reply:
The dataset is just 10 queries with two features and the model have 8 neurons.

Reply:
Doesn't matter. We need a unit test to help verify what MC Scorer does.